bash_aliases

```diff
@@ -40,9 +40,11 @@ function dl() {
           fi
         fi
       fi
+
+      cp ~/.daftlabs/hooks/require-pivotal-commit-msg.sh .git/hooks/commit-msg && chmod -R 755 .git/hooks
```

Comment:
it would be nice if this could be run from anywhere in the project and still work